### PR TITLE
Operation documentation

### PR DESCRIPTION
**Pull request by wiktork**
- [x] Do a pass over all the links to make sure they work correctly.

### CODE REVIEW

File: documentation/api/operations-delete.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Host Address
+
+The default host address for these routes is `https://localhost:52323`. This route is only available on the addresses configured via the `--urls` command line parameter and the `DOTNETMONITOR_URLS` environment variable.
+
+## Authentication
+
+Authentication is enforced for this route. See [Authentication](./../authentication.md) for further information.
+
+Allowed schemes:
+- `MonitorApiKey`
```
Comment (wiktork) [inline documentation/api/operations-delete.md:20]: @kelltrick I didn't want to change these before your changes go in.

File: documentation/api/dump.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ Allowed schemes:
 | Name | Type | Description | Content Type |
 |---|---|---|---|
 | 200 OK | stream | A managed dump of the process. | `application/octet-stream` |
+| 202 Accepted | | When specifying an egress provider, returns a Location of the operation to query for egress status. | |
```
[the remark this answers is not in the document]
Comment (jander-msft) [reply documentation/api/dump.md:46]: Whatever you change it to, update it across all of the docs.

File: documentation/api/operations-get.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+### Sample Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "resourceLocation":"https://example.blob.core.windows.net/dotnet-monitor/artifacts%2Fcore_20210721_062115",
+    "error":null,
+    "operationId":
```
Comment (jander-msft) [inline documentation/api/operations-get.md:50]: nit: Please put the value on the same line as the property and spaces after the colons.

File: documentation/api/operations.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Operations
+
+Operations are used to track long running operations in dotnet-monitor, specifically egressing data via egressProviders instead of directly to the client. This api is very similiar to [Azure asynchronous operations](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/async-operations#url-to-monitor-status).
+
+| Operation | Description |
+|---|---|
+| [List Operations](operations-list.md). | Lists all the operations and their current state. |
```
Comment (jander-msft) [inline documentation/api/operations.md:7]: nit: Remove the period after the end of link.

**Review by jander-msft — APPROVED** [review]

**Pull request merged**